src/api/buddies.ts

```diff
@@ -136,3 +143,21 @@ export function banBuddies(
       toBuddies,
     );
 }
+
+export const createFetchChunks = (
+  buddyIds: Array<string>,
+): Array<PollingParams> => {
+  const chunks = buddyIds.reduce<Array<string[]>>((resultArray, item, i) => {
+    const chunkIndex = Math.floor(i / 40);
+
+    if (!resultArray[chunkIndex]) {
+      resultArray[chunkIndex] = []; // start a new chunk
+    }
+
+    resultArray[chunkIndex].push(item);
+
+    return resultArray;
+  }, []);
+
+  return chunks.map(ch => ({ type: 'InitialMessages', buddyIds: ch }));
+};
```

Comment:
What is this?

Reply:
too complex for me. Can u explain and let's see how to make it easier to understand.

Reply:
```[...Array(Math.floor(10)).keys()].map(index => buddyIds.slice(index, (index + 1) * 40)).map(chunk => { some: 'lolwut', chunk } );```

Reply:
only maps : P

Reply:
![image](https://user-images.githubusercontent.com/34128180/146227007-ab29764d-89ff-4ed8-8f5a-9d9d04efb44e.png)
It looks better to you like this?

Reply:
yeah, but (index * 40, (index + 1) * 40)